server/src/models/user.js

```diff
@@ -0,0 +1,24 @@
+﻿import mongoose from 'mongoose';
+
+const userSchema = new mongoose.Schema({
+  username: {
+    type: String,
+    required: true
+  },
+  email: {
+    type: String,
+    required: true,
+    unique: true
+  },
+  password: {
+    type: String,
+    required: true
+  },
+  date: {
```

Comment:
This property name is ambiguous. In the response u used createdAt, I would keep the name createdAt in database as well

Reply:
ok